src/client/components/AddMeal.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from "react";
+import postData from "./postData";
+
+const minDate = () => {
+    const day = new Date();
+    let dd = day.getDate();
+    let mm = day.getMonth() + 1;
+    let hh = day.getHours()
+    let minutes = day.getMinutes()
+
+    const yyyy = day.getFullYear();
+    dd = dd < 10 ? '0' + dd : dd;
+    mm = mm < 10 ? '0' + mm : mm;
+    hh = hh < 10 ? '0' + hh : hh;
+    minutes = minutes < 10 ? '0' + minutes : minutes;
+    const today = yyyy + '-' + mm + '-' + dd + 'T' + hh + ':' + minutes
+    return today;
+}
+
+const AddMeal = () => {
+    const [title, setTitle] = useState("");
+    const [description, setDescription] = useState("");
+    const [location, setLocation] = useState("");
+    const [max_reservations, setMaxReservations] = useState("");
+    const [when, setWhen] = useState("");
+    const [price, setPrice] = useState("");
+
+
+
+    const setStatesEmpty = () => {
+        setTitle("")
+        setDescription("")
+        setLocation("")
+        setMaxReservations("")
+        setWhen("")
+        setPrice("")
+    }
+
+    function onSubmit(e) {
+        e.preventDefault();
+        const newMeal = {
+            title,
+            location,
+            description,
+            max_reservations,
+            when,
+            price
+        };
+
+        const response = postData('/api/meals', newMeal)
+        console.log(response);
+
+        if (response) {
+            alert(`Thank You, Your Meal : ${newMeal.title} Added`)
+        }
+        else {
+            throw new Error(response.status)
```

Comment:
What error are you checking for here and how are you handling it? What will happen if the Error is thrown?